domain-models-runtime/src/test/java/org/folio/rest/RestLauncherTest.java

```diff
@@ -0,0 +1,57 @@
+package org.folio.rest;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+
+public class RestLauncherTest {
+
+  private static final String DISABLE_METRICS = "-Dvertx.metrics.options.enabled=false";
+  private static final String ENABLE_METRICS = "-Dvertx.metrics.options.enabled=true";
+
+  @Test
+  public void testMetricsOptions01() {
```

Comment:
Can you avoid the number and use a descriptive method name for readability, for example `metricsIsDisabledByDefault`?